ruby/ql/lib/codeql/ruby/security/UrlRedirectCustomizations.qll

```diff
@@ -50,7 +50,9 @@ module UrlRedirect {
   /**
    * A source of remote user input, considered as a flow source.
    */
-  class RemoteFlowSourceAsSource extends Source, RemoteFlowSource { }
+  class HttpRequestInputAccessAsSource extends Source, Http::Server::RequestInputAccess {
```

Comment:
Could you explain this change? I would expect `Http::Server::RequestInputAccess` to be included in `RemoteFlowSource` and having URL redirects with input from other remote sources that are not `Http::Server::RequestInputAccess` would still be bad.